tests/utests/betting/create_matched_bet_tests.cpp

```diff
@@ -0,0 +1,242 @@
+#include <boost/test/unit_test.hpp>
```

Comment:
No need have these tests I believe. The only thing we need to check is that by_game_id_wincase index returns data in correct order.